business-registry-dashboard/app/components/Table/AffiliatedEntity/Action.vue

```diff
@@ -193,18 +194,25 @@ const handleApprovedNameRequestRenew = (item: Business): void => {
   }
 }
 
-const handleApprovedNameRequestChangeName = (item: Business, nrRequestActionCd: NrRequestActionCodes): void => {
-  if (!isModernizedEntity(item)) {
```

Comment:
This `isModernizedEntity` is the function that checks the Feature Flag. Removed now, no more FF check.